tests/post/steps/test_logs.py

```diff
@@ -19,10 +22,14 @@ def test_logs(host):
     'nginx-ingress-default-backend',
 ))
 
+# Constants
+POD_STATUS = 'Running'
```

Comment:
I think you don't need a global variable for this. Just add a variable in the function directly if you want